src/pretalx/schedule/exporters.py

```diff
@@ -179,15 +195,15 @@ def get_data(self, **kwargs):
                         "rooms": {
                             str(room["name"]): [
                                 {
+                                    "url": talk.submission.urls.public.full(),
                                     "id": talk.submission.id,
                                     "guid": talk.uuid,
-                                    "logo": talk.submission.urls.image,
                                     "date": talk.local_start.isoformat(),
                                     "start": talk.local_start.strftime("%H:%M"),
+                                    "logo": talk.submission.image_url or None,
```

Comment:
Why the move down, shouldn't ordering in JSON be irrelevant?

Reply:
yes, but it makes it mode human readable/easier to diff when using other tools/generators